packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/signingout.md

```diff
@@ -1,42 +1,21 @@
 This code assumes storing the access token after a successful sign-in flow:
 
 ```kotlin
+import com.okta.authfoundation.client.OidcClientResult
+import com.okta.authfoundation.credential.RevokeTokenType
+import com.okta.authfoundationbootstrap.CredentialBootstrap
+
 fun logout() {
-    viewModelScope.launch(Dispatchers.IO) {
-        try {
-            // First load a refresh token if one exists.
-            val refreshToken = Storage.tokens.refreshToken
-            if (refreshToken != null) {
-                // Revoking the refresh token also revokes the access token.
-                revokeToken("refresh_token", refreshToken)
-            } else {
-                revokeToken("access_token", Storage.tokens.accessToken)
+    viewModelScope.launch {
+        // Revoking the refresh token also revokes the access token.
+        when (val revokeResult = CredentialBootstrap.defaultCredential().revokeToken(RevokeTokenType.REFRESH_TOKEN)) {
```

Comment:
Is it possible to split kotlin code over two lines? This one runs out of the browser window.

```suggestion
        when (val revokeResult = CredentialBootstrap.defaultCredential()
            .revokeToken(RevokeTokenType.REFRESH_TOKEN)) {
```

Reply:
Did the best I could... in the next commit.